### PR TITLE
Travis: Build against more python versions

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@ language: python
 python:
   - "2.7"
   - "3.3"
+  - "3.4"
+  - "3.5"
+  - "pypy3"
```
Comment (twolfson) [inline .travis.yml]: Judging by the name `pypy3` is a major version. Should we lock down to minor version as we do with CPython? (e.g. `3.4`)

Comment (sils1297) [inline .travis.yml]: I am not aware that Travis supports that. PyPy 3 supports only the 3.2
specification right now with the important difference that it supports
unicode strings (so not supporting 3.2 but pypy3 is actually a useful
thing.)

Comment (twolfson) [inline .travis.yml]: Yea, it looks like Travis CI only does major version for PyPy.

http://docs.travis-ci.com/user/languages/python/#PyPy-Support

**Comment by twolfson** [comment]
Landing this PR now~

**Comment by twolfson** [comment]
This has been merged/released in `0.12.4`. Thanks for the PR!

**Comment by sils1297** [comment]
thanks for merging, would you have anything against us adding windows
builds for you? We're already testing your tool against windows in coala
and it works, it would probably be better if you catch your windows
problems before we hit them.

2015-11-16 9:05 GMT+01:00 Todd Wolfson <notifications@github.com>:

**Comment by twolfson** [comment]
I'm open to it but I typically avoid it unless the package is doing something OS specific (e.g. running a program, reading a hardcoded filepath from disk).

The reasoning is that the more versions/platforms that we test, the more likely something is to break in those external systems (e.g. what happens if Appveyor has a regression in their CI platform) which leads to more maintenance.

**Comment by sils1297** [comment]
sure then I'll not do it, as long as nothing bad happens with file paths we
all should be fine.

2015-11-16 9:31 GMT+01:00 Todd Wolfson <notifications@github.com>:

**Pull request merged**